webview_panels/src/modules/healthCheck/IssueList.tsx

```diff
@@ -0,0 +1,131 @@
+import { Card, CardTitle, CardBody, Button, Stack, Dropdown } from "@uicore";
+import classes from "./healthcheck.module.scss";
+import { Dispatch, useMemo, useState } from "react";
+import { panelLogger } from "@modules/logger";
+import { ArrowUpIcon, ArrowDownIcon } from "@assets/icons";
+import { ModelInsight, ProjectHealthcheck } from "./types";
+
+const IssueList = ({
+  projectHealthcheck,
+  setSeletedInsight,
+  setModalOpen,
+}: {
+  projectHealthcheck: ProjectHealthcheck;
+  setSeletedInsight: Dispatch<React.SetStateAction<ModelInsight | null>>;
+  setModalOpen: Dispatch<React.SetStateAction<boolean>>;
+}): JSX.Element => {
+  const [fileFilter, setFileFilter] = useState("");
+  const [insightTypeFilter, setInsightTypeFilter] = useState("");
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const { model_insights } = projectHealthcheck;
+  const [openAccordion, setOpenAccordion] = useState<Record<string, boolean>>(
+    {},
+  );
+  const insightFilenames = useMemo(() => {
+    return Object.entries(model_insights).map(([k, v]) => ({
+      label: v[0].original_file_path,
+      value: k,
+    }));
+  }, [model_insights]);
+  const insightTypes = useMemo(
+    () =>
+      Array.from(
+        new Set(
+          Object.values(model_insights)
+            .flat()
+            .map((x) => x.insight.type),
+        ),
+      ).map((x) => ({ label: x, value: x })),
+    [model_insights],
+  );
+  const filteredModelInsights = useMemo(() => {
+    const modelInsights: Record<string, ModelInsight[]> = {};
+    const insights = Object.values(model_insights)
+      .flat()
+      .filter(
+        (v) =>
+          (!fileFilter || v.unique_id === fileFilter) &&
+          (!insightTypeFilter || v.insight.type === insightTypeFilter),
+      );
+    for (const insight of insights) {
+      modelInsights[insight.unique_id] = modelInsights[insight.unique_id] || [];
+      modelInsights[insight.unique_id].push(insight);
+    }
+    return modelInsights;
+  }, [fileFilter, insightTypeFilter, model_insights]);
+  panelLogger.log(filteredModelInsights);
+  return (
+    <Card className={classes.container}>
+      <CardTitle>
+        <Stack className="align-items-center">
+          <div className="fs-5">Issues</div>
```

Comment:
pls use [typography](https://github.com/AltimateAI/vscode-dbt-power-user/blob/master/webview_panels/src/uiCore/theme.scss#L106) instead of fs classes, so we could have consistent styling across components